error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -168,9 +168,8 @@ Stream<R> after(Stream<T> stream, Function<? super S, ? extends Stream<? extends
    * Where possible, clarify that a mapping operation will be applied only to a single stream
    * element.
    */
-  // XXX: Implement a similar rule for `.findAny()`. For parallel streams this wouldn't be quite the
-  // same, so such a rule requires a `Matcher` that heuristically identifies `Stream` expressions
-  // with deterministic order.
+  // XXX: Consider whether to have a similar rule for `.findAny()`. For parallel streams it
+  // wouldn't be quite the same....
```

Comment:
```suggestion
  // wouldn't be quite the same...
```
But did you even intend to drop the rest of the comment?